drivers/sensor/bme280/bme280.c

```diff
@@ -254,7 +258,7 @@ static int bme280_channel_get(const struct device *dev,
 		val->val2 = (((data->comp_humidity & 0x3ff) * 1000U * 1000U) >> 10);
 		break;
 	default:
-		return -EINVAL;
+		return -ENOTSUP;
```

Comment:
Please create a separate commit with that change.

Reply:
For consistency I added this change first, so that we don't have a commit that may return different error codes on the same error.

Reply:
You have a typo in the commit title -> `ENOSUP`.

Reply:
Thanks, good catch. Fixed